idea/tests/test/org/jetbrains/kotlin/idea/caches/resolve/IdeCompiledLightClassTestGenerated.java

```diff
@@ -21,7 +21,7 @@
 @TestRoot("idea/tests")
 @TestDataPath("$CONTENT_ROOT")
 @RunWith(JUnit3RunnerWithInners.class)
-@TestMetadata("../../../intellij/out/kotlinc-testdata/testData/compiler/testData/asJava/lightClasses")
+@TestMetadata("../../intellij/out/kotlinc-testdata/testData/compiler/testData/asJava/lightClasses")
```

Comment:
There an inconsistency with test testdata paths in intellij-kotlin on github and in our internal repository caused by different directory structure in repositories. That's why it always regenerates tests :) It should be fixed soon